technical.js

```diff
@@ -74,7 +74,7 @@ module.exports = {
         });
         prev_ema = sum / length;
       }
-      const multiplier = 2 / (length + 1);
```

Comment:
I might be missing something but what's the reasoning for changing these into let vs. const?

Reply:
I agree that it could be const, but better to use const for constant values and var/let for variables even if they never changed. This is practice of good coding. For example:
const x = 10;
let y = x / 2;